### PR TITLE
Re-animate Rebel Hummingbird

### PR DESCRIPTION
**Pull request by jcbjoe**
- Benches on the side have been re-added.
- Garage and shop

### CODE REVIEW

**Comment by Jawshy** [comment]
[Resolves](https://help.github.com/articles/closing-issues-via-commit-messages/) https://github.com/AsYetUntitled/Framework/issues/43.

File: Altis_Life.Altis/core/shops/fn_vehicleShopBuy.sqf [hunk 1]
```diff
@@ -113,6 +113,9 @@ switch (playerSide) do {
         [_vehicle,"cop_offroad",true] spawn life_fnc_vehicleAnimate;
     };
     case civilian: {
+        if ((life_veh_shop select 0) isEqualTo "reb_car" && {_className == "B_Heli_Light_01_stripped_F"}) then {
```
Comment (tkcjesse) [inline Altis_Life.Altis/core/shops/fn_vehicleShopBuy.sqf]: @jcbjoe can you do isEqualTo here:  ```_className == "B_Heli_Light_01_stripped_F"```

**Comment by jcbjoe** [comment]
Yeah ill get that done. Thanks :)

**Comment by Jawshy** [comment]
What's odd is that on the current build (https://github.com/AsYetUntitled/Framework/commit/57c2d8baa2627b030748bcccd3a00ef07b1976ab) the rebel Hummingbird already has benches; both when purchased and when retrieved from the garage.

**Comment by tkcjesse** [comment]
Yeah, nevermind. I was doing some cocaine and didn't realize some shit. The way it is currently is fine. It's when you're stupid and change stuff up.

**Pull request closed without merging**